src/extension.ts

```diff
@@ -181,10 +189,16 @@ async function getWebviewContent(
         vscode.Uri.file(path.join(context.extensionPath, "static", fileName))
       )
       .toString();
+  
+  // replace ${name} with given variable
```

Comment:
Is this comment up to date? To be honest I have no idea what `${name}` it refers to.